portfolio/src/main/webapp/about-me.html

```diff
@@ -26,6 +26,48 @@ <h1>My Portfolio</h1>
                 </div>
             </div>
         </div>
+
+        <div class="content">
+            <div class="two-columns">
+                <div class="about-me-img">
+                    <img src="images/about-me-img1.jpeg" alt="profile-pic.jpeg" class="circular-portrait">
+                </div>
+                <div class="about-me-content">
+                    <div class="three-rows">
+                        <div class="row-element">
+                            <h2>This is me!</h2>
+                            <br>
+                            Hello! My name is Chance Lau, and I study computer science at the
+                            University of California - Davis.
+                            <br>A list of 10 things about me:</br>
```

Comment:
I think `<br>` should just be used to force a line break. Where you have actual paragraphs, it's better to use `<p>`...`</p>`.

You could also consider using `<ol>` for your numbered list below.

Reply:
+1 to ol

Reply:
Done!